WooCommerce/Classes/ViewRelated/Orders/Order Details/Issue Refunds/IssueRefundViewController.swift

```diff
@@ -163,6 +163,16 @@ private extension IssueRefundViewController {
         headerStackView.alignment = headerStackView.axis == .vertical ? .center : .fill
         headerStackView.spacing = 8
     }
+
+    func updateButtonState(enabled: Bool, showActivityIndicator: Bool) {
+        nextButton.isEnabled = enabled && !showActivityIndicator
+        switch showActivityIndicator {
+        case true:
+            nextButton.showActivityIndicator()
+        case false:
+            nextButton.hideActivityIndicator()
+        }
```

Comment:
nit: using just `if else` saves one line 😄 
```suggestion
        if showActivityIndicator {
            nextButton.showActivityIndicator()
        } else {
            nextButton.hideActivityIndicator()
        }
```

Reply:
Very true! I am trying to reach for `switch` more, because it's often more expressive, but this is not one of those times.